requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -18,86 +18,105 @@ The 'full' onboarding experience is delivered across some 'sub' campaigns, below
-    1. If `Yes`, do nothing. (See Feature flag onboarding flow)
-    2. If `No`, assign to random cohort branch. 
-        1. 50% receive Feature Flag Upsell email 1
-        2. 50% recieve Feature Flag Upsell email 2
-12. Wait 5 days. 
-13. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
-14. Check if the user is in the `Subscribers to Surveys` segment
-    1. If `Yes`, do nothing. (See Survey onboarding flow)
+4. Wait 1 week.
+5. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+6. Check if the user is in the `Subscribers to Product Analytics` segment
+    1. If `No`, send Analytics Upsell
+    2. If `Yes`, check if the user is in the `Subscribers to Session Replays` segment
+        1. If `No`, send Replay Upsell email
+        2. If `Yes`, check if the user is in the `Subscribers to Feature Flags` segment
+            1. If `No`, send Feature Flags Upsell email.
+            2. If `Yes`, check if the user is in the `Subscribers to Surveys` segment
+                1. If `No`, send Surveys Upsell email
+                2. If `Yes`, send email about enabling beta features
+7. Wait 1 week. 
+8. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+9. Check if the user is in the `Subscribers to Session Replays` segment
+    1. If `No`, send Replay Upsell email
+    2. If `Yes`, check if the user is in the `Subscribers to Feature Flags` segment
+        1. If `No`, send Feature Flags Upsell email.
+        2. If `Yes`, check if the user is in the `Subscribers to Surveys` segment
+            1. If `No`, send Surveys Upsell email
+            2. If `Yes`, send email inviting user to join the PostHog community
+10. Wait 1 week. 
+11. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+12. Check if the user is in the `Subscribers to Feature Flags` segment
+    1. If `No`, send Feature Flags Upsell email.
+    2. If `Yes`, check if the user is in the `Subscribers to Surveys` segment
+        1. If `No`, send Surveys Upsell email
+        2. If `Yes`, send email email about dashboard and survey templates 
+13. Wait 1 week. 
+14. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
+15. Check if the user is in the `Subscribers to Surveys` segment
+    1. If `Yes`, send an email pushing about building like PostHog
     2. If `No`, send Survey Upsell email
-15. Add `completed_onboarding_emails: true` to user. 
+16. Add `completed_onboarding_emails: true` to user 
+
+### Product analytics onboarding flow
+**Campaign in Customer.io:** Product Analytics onboarding
+
+This campaign triggers when a user enters the `Subscribers to Product Analytics` segment, provided they also have a valid email address. 
+
+1. Wait 1 day, and until a weekday between 9AM and 12PM in the users' time zone (UTC fallback)
+2. Check if user is in the `Subscribers to Group Analytics` segment
+    1. If `Yes`, send Product Onboarding Email
```

Comment:
<img width="963" alt="Screenshot 2023-11-14 at 16 18 21" src="https://github.com/PostHog/meta/assets/84011561/bc35938c-5d15-4358-a038-a69dcf24625d">

Reply:
Last sentence is a bit word, may change to: "and we've got plenty of guides to inspire what's next"